src/lib.rs

```diff
@@ -59,6 +59,12 @@
 // Disallow warnings in examples.
 #![doc(test(attr(deny(warnings))))]
 
+#[cfg(test)]
+extern crate quickcheck;
+#[cfg(test)]
+#[macro_use(quickcheck)]
+extern crate quickcheck_macros;
```

Comment:
FYI: since rust 2018 this is no longer required.

Reply:
`#[quickcheck]` attribute won't work without it: https://github.com/BurntSushi/quickcheck/blob/master/README.md#the-quickcheck-attribute

However if you don't want any new dependencies, this will go away.